include/GL/glui.h

```diff
@@ -34,11 +34,23 @@
 #ifndef GLUI_GLUI_H
 #define GLUI_GLUI_H
 
-// Having stdlib here first fixes some 'exit() redefined' errors on MSVC.NET
-// that come from old GLUT headers.
-#include <cstdlib>
+//Note: GCC is not required for this feature. It's just an indicator. 
+#ifdef __GNUC__
+//GLUI_GLX uses glXGetCurrentDisplay to find GLUT's chosen X display.
+//It will #include GL/glx.h and X11/Xlib.h. This covers all desktops.
+#ifndef GLUI_GLX
+#define GLUI_GLX
+#endif
+#endif
+
+#if defined(GLUT_API_VERSION) //|| defined(?)
```

Comment:
I filed a PR to move the `glut.h` inclusion from the GLUI header here:  https://github.com/libglui/glui/pull/76